[readline-unix ] build failure

<!-- ⚠ Please use the generated issue template instead to report a port build failure. ⚠ -->
<!-- ⁉ You can find its location at the end of the console output. ⁉ -->

**Host Environment**

- OS: wsl2 ubuntu 22.04
- Compiler: gcc 11.3

**To Reproduce**

Steps to reproduce the behavior:

`./vcpkg install readline`

**Failure logs**

```
Building readline-unix[core]:x64-linux...
-- Downloading https://ftpmirror.gnu.org/gnu/readline/readline-8.2.tar.gz;https://ftp.gnu.org/gnu/readline/readline-8.2.tar.gz -> readline-8.2.tar.gz...
-- Extracting source /home/tal/vcpkg/downloads/readline-8.2.tar.gz
-- Using source at /home/tal/vcpkg/buildtrees/readline-unix/src/readline-8-e421b8215c.clean
-- Found external ninja('1.11.1.git.kitware.jobserver-1').
-- Getting CMake variables for x64-linux-dbg
-- Getting CMake variables for x64-linux-rel
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
-- Installing x64-linux-dbg
-- Building x64-linux-rel
-- Installing x64-linux-rel
-- Fixing pkgconfig file: /home/tal/vcpkg/packages/readline-unix_x64-linux/lib/pkgconfig/history.pc
-- Fixing pkgconfig file: /home/tal/vcpkg/packages/readline-unix_x64-linux/lib/pkgconfig/readline.pc
-- Fixing pkgconfig file: /home/tal/vcpkg/packages/readline-unix_x64-linux/debug/lib/pkgconfig/history.pc
-- Fixing pkgconfig file: /home/tal/vcpkg/packages/readline-unix_x64-linux/debug/lib/pkgconfig/readline.pc
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:135 (message):
  /bin/pkg-config --exists readline failed with error code: 1

      ENV{PKG_CONFIG_PATH}: "/home/tal/vcpkg/packages/readline-unix_x64-linux/debug/lib/pkgconfig:/home/tal/vcpkg/packages/readline-unix_x64-linux/share/pkgconfig:/home/tal/vcpkg/installed/x64-linux/debug/lib/pkgconfig:/home/tal/vcpkg/installed/x64-linux/share/pkgconfig"
      output: Package ncurses was not found in the pkg-config search path.

  Perhaps you should add the directory containing `ncurses.pc'

  to the PKG_CONFIG_PATH environment variable

  Package 'ncurses', required by 'readline', not found
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:200 (z_vcpkg_fixup_pkgconfig_check_files)
  ports/readline-unix/portfile.cmake:26 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:147 (include)
```
- Cut and paste the appropriate build messages from the console output.
- Attach ***every failure log file*** mentioned in the console output.



<!-- ‼ **The console output by itself is not enough to investigate a build failure.** ‼ -->

**Additional context**

Add any other context about the problem here, such as what you have already tried to resolve the issue.

It looking for `ncurses.pc` but in the installed debug there is: `ncurses_g.pc`
It need to rename the `ncurses.pc` in one of the ports to match. What is the correct name?

--- a/ports/ncurses/portfile.cmake
+++ b/ports/ncurses/portfile.cmake
@@ -17,9 +17,9 @@ vcpkg_list(SET OPTIONS)
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
     list(APPEND OPTIONS
-        --with-shared
         --with-cxx-shared
-        --without-normal
+        --with-shared    # "lib model"
+        --without-normal # "lib model"
     )
 endif()
 
@@ -46,17 +46,12 @@ vcpkg_configure_make(
         --disable-db-install
         --enable-pc-files
         --without-ada
+        --without-debug # "lib model"
         --without-manpages
         --without-progs
         --without-tack
         --without-tests
         --with-pkg-config-libdir=libdir
-    OPTIONS_DEBUG
-        --with-debug
-        --without-normal
-    OPTIONS_RELEASE
-        --without-debug
-        --with-normal
 )
 vcpkg_install_make()
 

--- a/ports/ncurses/vcpkg.json
+++ b/ports/ncurses/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "ncurses",
   "version": "6.4",
+  "port-version": 1,
   "description": "Free software emulation of curses in System V Release 4.0, and more",
   "homepage": "https://invisible-island.net/ncurses/announce.html",
   "license": "MIT",

--- a/scripts/test_ports/cmake-user/vcpkg.json
+++ b/scripts/test_ports/cmake-user/vcpkg.json
@@ -134,6 +134,11 @@
           "platform": "!uwp & !mingw"
         },
         {
+          "$package": "Curses",
+          "name": "ncurses",
+          "platform": "!windows | mingw"
+        },
+        {
           "$package": "PhysFS",
           "name": "physfs"
         },

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5350,7 +5350,7 @@
     },
     "ncurses": {
       "baseline": "6.4",
-      "port-version": 0
+      "port-version": 1
     },
     "neargye-semver": {
       "baseline": "0.3.0",

--- a/versions/n-/ncurses.json
+++ b/versions/n-/ncurses.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "faaf5fff20ee04db0b33a595cd37f41b89c840d6",
+      "version": "6.4",
+      "port-version": 1
+    },
+    {
       "git-tree": "c557b7daa2f3a87a54e037e7b984ad8a661c5903",
       "version": "6.4",
       "port-version": 0

